NashTeamAlpha/BangazonWeb
# Shopping Cart View Model

## Pull Request Description

Created the ShoppingCartViewModel.cs in the ViewModels folder.
The ShoppingCartViewModel should pass the view a list of products(all products on active order),
and a list of PaymentTypes (All PaymentTypes of the active user)
## Steps to Test

Check that an IEnumerable of type products is being passed
Check that an IEnumerable of type PaymentTypes is being passed
## Link to Feature Ticket
#77
